README.md

```diff
@@ -19,21 +19,7 @@ cd frontend-burgernabije-besluitendatabank
 npm install
 ```
 
-From there, you can use `npm run dev:proxy`, or any of the following npm scripts:
-
-
-|  npm run...  |  Description |
```

Comment:
Wait, was this intentional? Merge conflict whoopsie?

Reply:
I am my own biggest enemy :(